### ISSUE TITLE
The guide shows to force delegation for a vote account but it seems not working

### ISSUE THREAD

**Issue by ochaloup** [mvines/validator-Identity-transition-demo#1]
Hi @mvines ,

I hope you don't mind I open this issue in rather stale of a discussion. At first, thank you for this guide. It's great. It helped me to start working with the local cluster of Solana. I would create a PR for this issue to adjust the guides to work for the current Solana master (1.11.x) if it you consider it worthy and right.

**What does not work for me**

Running the command 

```
solana -ul delegate-stake stake-account.json vote-account.json --force
```
https://github.com/mvines/validator-Identity-transition-demo/blob/master/README.md?plain=1#L48

There is no `--force` argument in CLI to force the of the non-active vote account. At least I understand the error message in way the vote account needs to be considered as "active" by cluster.

```
Bad parameter: Unable to delegate. Vote account has no root slot
```

```
solana vote-account /tmp/validator-ledger/vote-account.json
Account Balance: 0.02685864 SOL
Validator Identity: FjDhhFfvp11HpHf4vBwPyaJShgR2kkeb9MLZ9ZddzaiE
Vote Authority: {1: "FjDhhFfvp11HpHf4vBwPyaJShgR2kkeb9MLZ9ZddzaiE"}
Withdraw Authority: FcrxmqfFNE4Ps7HgHqYRJ67fmaxVaU5LWZRqmHJwK23A
Credits: 0
Commission: 100%
Root Slot: ~
Recent Timestamp: 1970-01-01T00:00:00Z from slot 0
```

What helps me is to start the `solana-validator` first with the already created vote-account. At time the validator catchs up the vote account starts to be available for delegating the stake.

Another thing that does not work (and I'm not sure if it's an issue or my wrong or it should be so) it's the `--vote-account` parameter that does not allow me to pass just the file but I need to go with publickey.

```
# not working
--vote-account vote-account.json

# working
--vote-account `solana-keygen pubkey vote-account.json`
```
https://github.com/mvines/validator-Identity-transition-demo/blob/master/README.md?plain=1#L71

**Additional question**

Anyway, one more question. I struggle time to time that the started validator is not capable to catchs up with the test validator and it falls behind forever. I consider that my laptop is slow enough to not allow multiple validators to run at required pace. I would like to know if there is a chance for me to somehow make the PoH/communication processing lighter/slower for the local testing may absorb multiple validators?

```
solana catchup --our-localhost 10999
# ...our node is falling behind at ~2.4 slots/second...
```

Thank you for any comment.

**Comment by mvines** [comment]
> There is no `--force` argument in CLI to force the of the non-active vote account

hmm, it worked for me just now:
```
$ solana -ul delegate-stake stake-account.json vote-account.json --force
--force supplied, ignoring: Bad parameter: Unable to delegate. Vote account has no root slot

Signature: 3tkXiDtCy2YW6c8xTenX96riQvKchCd438PukFoZ1Lo6B2GZFZQeK3W4SDBrPmTm13wcovDQJhorFYfS1ayDQdVG

$ solana --version
solana-cli 1.12.0 (src:ecda3bec; feat:3605932003)
```

> Another thing that does not work (and I'm not sure if it's an issue or my wrong or it should be so) it's the --vote-account parameter that does not allow me to pass just the file but I need to go with publickey.

Works here. Seems like you're using some perhaps very old version of the cli?

> Anyway, one more question. I struggle time to time that the started validator is not capable to catchs up with the test validator and it falls behind forever. I consider that my laptop is slow enough to not allow multiple validators to run at required pace. I would like to know if there is a chance for me to somehow make the PoH/communication processing lighter/slower for the local testing may absorb multiple validators?

The test-validator is already configured to use "sleepy PoH" to lighten the load. Sounds like you need a new laptop!

**Comment by ochaloup** [comment]
> > There is no `--force` argument in CLI to force the of the non-active vote account
> 
> hmm, it worked for me just now:

Oh, I had to be blind. The force flag has just worked for me now. The same with the vote account.
The issue with the vote account could be that I mixed an old CLI installed and the validator built from master. It general, problem exists between chair and keyboard.
I'm sorry for bothering you with this.

> The test-validator is already configured to use "sleepy PoH" to lighten the load. Sounds like you need a new laptop!

Thank you for pointing me for the `hashes_per_tick` configuration. I do understand a bit more on what's happening now.
I'm still not getting why my laptop is so clumsy to manage the most one more validator on top of the test one (any other one just falls behind). There could be different reasons. I will try to understand the code a bit more to evaluate what's wrong with me :-)
Thanks again for the guidance.
